origin-dapp/public/schemas/forSale-antiques_1.0.0.json

```diff
@@ -33,14 +33,14 @@
       "minLength": 10,
       "maxLength": 1024
     },
-    "price": {
-      "type": "number",
-      "title": "schema.priceInETH"
-    },
     "unitsTotal": {
       "type": "integer",
       "title": "schema.unitsTotal"
     },
+    "price": {
```

Comment:
The order of fields in the schema files also dictates the order of fields in the `create-listing` form. There was a UX decision that it is more natural to the user to fill out number of units (quantity) before filling out the price because the label on the price field changes from `price` to `price per unit` when quantity is bigger than 1.